src/components/HomePageComponents/FeatFundraiser.js

```diff
@@ -6,7 +6,7 @@ const HomePageFeatFundraiser = (props) => {
   return (
     <div>
         <div className='text-5xl pb-4 font-HomePageHeading font-semibold'>
-                    Today's Featured Fundraiser
+                    Featured Fundraiser
```

Comment:
I don't think you were in class yet when the professor gave us this comment, but for the featured fundraiser is there a way to make the description text for the fundraiser only partially visible unless the user expands the text or goes to that individual fundraiser page? He said it displaying all the text for a fundraiser was too much or something like that

Reply:
I'll look into it!  If you have a link to an example, let me know.  Trying to picture it, makes sense.

Reply:
<img width="712" alt="image" src="https://user-images.githubusercontent.com/77643903/226496389-ec4f2710-65e6-4017-9a3d-d10f2457aaa1.png">

Reply:
Maybe something like that?

Reply:
@frogsupport,  I think the idea is cool!  Extensive work, not absolutely necessary.  But, if still interested, we could probably find a package for it possibly. Something to look into.